Windows Compatibility

hissw was written and tested on macOS and Linux. No tests have been performed on Windows, but it appears that it does not seem to work there. 

As noted by @schonfsj in #7, 

> I actually can't get the functionality to work, but that's because I'm running a weird hybrid system with IDL running in windows and python running in a WSL linux environment. So it's a compatibility problem of my own creation.

It's not clear what changes would need to be made to the subprocess calls to get this working.

I couldn't get it to work on Windows 10.

When I installed (trying both through `pip` and direct cloning of the repository), it did not create the directory `$HOME/.hissw/`.  Trying to run pytest does not work as a result.

```
(base) C:\Users\reep\Documents\hissw>pytest --ssw-home="C:\ssw" --idl-home="C:\Program Files\Harris\IDL89\bin\bin.x86_64"
============================================ test session starts ============================================
platform win32 -- Python 3.11.5, pytest-7.4.0, pluggy-1.3.0
rootdir: C:\Users\reep\Documents\hissw
plugins: anyio-3.5.0, asdf-3.0.1
collected 0 items / 1 error
```
...
```
   from .read_config import defaults
hissw\read_config.py:10: in <module>
    defaults['hissw_home'] = os.path.join(os.environ['HOME'], '.hissw')
<frozen os>:679: in __getitem__
    ???
E   KeyError: 'HOME'
========================================== short test summary info ==========================================
ERROR hissw/tests/test_hissw.py - KeyError: 'HOME'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================= 1 error in 0.44s ==============================================
```

--- a/hissw/read_config.py
+++ b/hissw/read_config.py
@@ -3,14 +3,18 @@ Read some default options if possible
 """
 
 import os
+import pathlib
 import configparser
+
+HISSW_DIR = pathlib.Path.home() / '.hissw'
+HISSW_RC = HISSW_DIR / 'hisswrc'
 
 
 defaults = {}
-defaults['hissw_home'] = os.path.join(os.environ['HOME'], '.hissw')
-if os.path.isfile(os.path.join(os.environ['HOME'], '.hissw', 'hisswrc')):
+defaults['hissw_home'] = HISSW_DIR
+if HISSW_RC.is_file():
     config = configparser.ConfigParser()
-    config.read(os.path.join(os.environ['HOME'], '.hissw', 'hisswrc'))
+    config.read(HISSW_RC)
     for key in config['hissw']:
         defaults[key] = config['hissw'][key]
 else:

